static/js/ui_init.js

```diff
@@ -103,13 +103,35 @@ import * as user_status_ui from "./user_status_ui";
    because we want to reserve space for the email address.  This avoids
    things jumping around slightly when the email address is shown. */
 
+export function animate(emoji) {
```

Comment:
Maybe we want to name these variables `$emoji_elt`?

Reply:
I changed to `$emoji`.

Does `elt` stand for `element`? I've seen the abbreviation a few times before.